app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -1143,7 +1131,7 @@
 					"earliest": "-24h",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights \n| stats count by application\n| eval application = case(\n      application==\"malware-detection\", \"Malware\",\n      !isnull(application), upper(substr(application,1,1)).substr(application,2)\n      )\n| rename application as Application, count as Count"
+				"query": "index=redhatinsights \n| stats count by application\n| eval application = case(\n      application==\"malware-detection\", \"Malware\",\n      application==\"resource-optimization\", \"ROS\",\n      !isnull(application), upper(substr(application,1,1)).substr(application,2)\n      )\n| rename application as Application, count as Count"
```

Comment:
Why is the ROS added here?

Reply:
Because the original name is too long, and we'll have the "split" behavior (eg: Resour...zation) on  the chart so, on Friday standup we decide to move to the ROS alias, to not have this behavior on the chart.